package.json

```diff
@@ -28,6 +28,7 @@
         "example": "npx tsx ./examples/lib/executeExample.ts"
     },
     "dependencies": {
+        "@balancer-labs/balancer-maths": "^0.0.23",
```

Comment:
Maybe adding this dependency altered some lock file version of biome which is why the Lint check is angry?

https://github.com/balancer/b-sdk/actions/runs/13619113723/job/38066048856#step:4:7

Reply:
See comment below.